spec/imsc-hrm.html

```diff
@@ -764,25 +762,19 @@ <h2>Paint Text</h2>
       depending on its position in a word.
       Some scripts require combining marks or use a sequence of code points to form a glyph.
       Cases exist where a given sequence of code points can have different glyph representations depending on context.
-      This complexity is accounted for by reducing the performance of the glyph buffer
+      This complexity is accounted for by reducing the performance of the Glyph Cache
       for scripts where a one-to-one mapping is not the general rule (see GCpy below).
       </p>
 
       <p>For each <a>glyph</a> associated with a <a>character</a> in a <a>presented region</a> of <a>Intermediate Synchronic Document</a>
       E<sub>n</sub>, the Presentation Compositor:</p>
 
       <ul>
-        <li>if an identical <a>glyph</a> is present in Glyph Buffer G<sub>n</sub>, copies the <a>glyph</a> from Glyph Buffer
-        G<sub>n</sub> to the Back Buffer using the Glyph Copier; or
-        </li>
+        <li>if an identical <a>glyph</a> is present in Glyph Back Buffer, copies the <a>glyph</a> from Glyph Cache
```

Comment:
Should this read "is present in Glyph Cache" rather than "is present in Glyph Back Buffer"?

Reply:
See https://github.com/w3c/imsc-hrm/pull/55/commits/8be035878a1943580b190e2187d43c0698ac22a3